shared/translations/languages/de.json

```diff
@@ -683,7 +695,9 @@
     "Swapping Token": "Token-Tausch",
     "Swapping {{amountA}} {{symbolA}} to {{amountB}} {{symbolB}}": "Tausch von {{amountA}} {{symbolA}} gegen {{amountB}} {{symbolB}}",
     "Slippage tolerance": "Slippage-Toleranz",
-    "Estimated to receive": "Geschätzter Betrag"
+    "Estimated to receive": "Geschätzter Betrag",
+    "Swap on future block {{amountA}} {{fromTokenDisplaySymbol}} to {{toTokenDisplaySymbol}} on oracle price {{percentageChange}}": "Tauschen auf zukünftigem Block {{amountA}} {{fromTokenDisplaySymbol}} => {{toTokenDisplaySymbol}} zum zukünftigem Preisorakelwert {{percentageChange}}",
```

Comment:
"Swap on future block {{amountA}} {{fromTokenDisplaySymbol}} to {{toTokenDisplaySymbol}} on oracle price {{percentageChange}}": "Tauschen auf zukünftigem Block von {{amountA}} {{fromTokenDisplaySymbol}} => {{toTokenDisplaySymbol}} zum zukünftigem Preisorakelwert {{percentageChange}}"